src/ucs/vfs/base/vfs_obj.c

```diff
@@ -617,6 +691,28 @@ ucs_vfs_path_list_dir(const char *path, ucs_vfs_list_dir_cb_t dir_cb, void *arg)
     return status;
 }
 
+ucs_status_t ucs_vfs_path_get_link(const char *path, ucs_string_buffer_t *strb)
+{
+    ucs_vfs_node_t *node;
+    ucs_status_t status;
+
+    ucs_spin_lock(&ucs_vfs_obj_context.lock);
+
+    node = ucs_vfs_node_find_by_path(path);
+    if (!ucs_vfs_check_node(node, UCS_VFS_NODE_TYPE_SYM_LINK)) {
+        status = UCS_ERR_NO_ELEM;
+        goto out_unlock;
+    }
+
+    ucs_vfs_get_link_path(node, strb);
```

Comment:
how does it work if the target of the link was removed and it's NULL? how will it look on the filesystem?\
maybe need a return value for "broken link" instead of returning an empty string?

Reply:
Updated the logic here.
Removed link nodes, if target node is removed; to w/a possibility of incorrect links creation.